### PR TITLE
MC-40750: QPT Landing Page

### PR DESCRIPTION
**Pull request by viktym**
## Purpose of this pull request

- Added warning about applying a large number of patches
- Added links on QPT Landing Page

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/project/project-patch.html
- https://devdocs.magento.com/quality-patches/tool.html
- https://devdocs.magento.com/quality-patches/usage.html

## Links to Magento source code

-  ...

### CODE REVIEW

**Review by BarnyShergold — APPROVED** [review]

File: src/cloud/project/project-patch.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ functional_areas:
 -  {{ site.data.var.mcp-prod }} delivers required patches with critical fixes
 -  Quality Patches deliver optional, low-impact quality fixes as [individual patches]({{ site.baseurl }}/release/policy/#individual-patch), which do not contain backward incompatible changes.
 
+See [patches available in the Quality Patches Tool]({{ site.baseurl }}/quality-patches/tool.html) to review a full list of released patches.
```
Comment (hguthrie) [inline src/cloud/project/project-patch.md:14]: Actually, the link on line 9 above is redirecting to the link you inserted here. I would suggest you update the Quality Patches Tool link in line 9 and then you don't have to add this line here. And we get rid of a redirect 😄

Comment (viktym) [reply src/cloud/project/project-patch.md:14]: ?? 
in Line 9 I see links on repositories 
```
[{{ site.data.var.mcp-prod }}](https://github.com/magento/magento-cloud-patches) and the [Quality Patches Tool](https://github.com/magento/quality-patches) deliver patches to your installed {{site.data.var.ee}} application.
```
Those links are totally different.
It's an internal PR, can we have it merged ASAP?
If anything else is required from me please ping me in a slack @tymchyns 
Thank you

Comment (hguthrie) [reply src/cloud/project/project-patch.md:14]: It is a different URL, but if you click on it from the topic page it redirects now to the URL you are trying to add.

**Review by hguthrie — APPROVED** [review]
No worries, I'll go ahead and fix it on my end.
😄

**Comment by hguthrie** [comment]
@magento import pr to magento-devdocs/devdocs

**Comment by m2-github-services** [comment]
@hguthrie the Pull Request is successfully imported.

**Comment by jeff-matthews** [comment]
@viktym, we can process internal PRs faster if you open them against the [magento-commerce/devdocs](https://github.com/magento-commerce/devdocs) repo.

**Pull request merged**